RevenueCatUI/UIKit/PaywallViewController.swift

```diff
@@ -59,57 +131,10 @@ public class PaywallViewController: UIViewController {
         fatalError("init(coder:) has not been implemented")
     }
 
-    private lazy var hostingController: UIHostingController<some View> = {
-        let purchaseHandler = PurchaseHandler()
-        purchaseHandler.refreshSubscriptions = refreshSubscriptions
-        let view = PaywallView(offering: self.offering,
-                               offeringSelection: self.offeringSelection,
-                               customerInfo: nil,
-                               mode: self.mode,
-                               displayCloseButton: self.displayCloseButton,
-                               introEligibility: nil,
-                               purchaseHandler: purchaseHandler)
-            .onPurchaseCompleted { [weak self] transaction, customerInfo in
-                guard let self = self else { return }
-                self.delegate?.paywallViewController?(self, didFinishPurchasingWith: customerInfo)
-                self.delegate?.paywallViewController?(self,
-                                                      didFinishPurchasingWith: customerInfo,
-                                                      transaction: transaction)
-            }
-            .onRestoreCompleted { [weak self] customerInfo in
-                guard let self = self else { return }
-                self.delegate?.paywallViewController?(self, didFinishRestoringWith: customerInfo)
-            }
-            .onSizeChange { [weak self] in
-                guard let self = self else { return }
-                self.delegate?.paywallViewController?(self, didChangeSizeTo: $0)
-            }
-            .onPurchaseInitiated { [weak self] selectedPackage in
-                guard let self = self else { return }
-                self.delegate?.paywallViewController?(self, didInitiatePurchaseWithSelectedPackage: selectedPackage)
```

Comment:
this was removed??

Reply:
The creation has been moved inside `createHostingController()`
Our `.onPurchaseInitiated` is not needed anymore since they have been added `onPurchaseStarted` that does the same thing.

Reply:
cool 👍